### DISCUSSION TITLE
Where can pre-built AppImage files be found?

### DISCUSSION THREAD

**Discussion by eliasdaler** [vgmtrans/vgmtrans#338] [General]
Hello.
README links to a Github Actions CI job and I can't find any artifacts attached to them.
Is there anywhere I can find them without building the project myself?

**Comment by admin** [comment]
Sorry for the delay in replying.
You can always download the latest from [here](https://nightly.link/vgmtrans/vgmtrans/workflows/build/master). A link is also present in [our homepage](https://vgmtrans.github.io/vgmtrans).

**Reply by eliasdaler** [reply]
Thanks. I think it will be good to link to this page in the README.
